tests/tests_rf/test_binding_fsm.py

```diff
@@ -54,6 +54,8 @@
 ITHO__ = "itho"
 NUAIRE = "nuaire"
 ORCON_ = "orcon"
+VASCO_ = "vasco"
+CLIMARAD = "climarad"
 
 TEST_SUITE_300 = [
     # {  # THM to CTL: FIXME: affirm is I|1FC9|07, not I|1FC9|00
```

Comment:
You need to add some test bindings of these systems. Look at **./tests/bindings/hvac/***

Reply:
Bindings are further down the To Do list. Have to climb on top of our bathroom to push a button on the MiniBox...
Took these lines out of this PR.